docs/features/logging.rst

```diff
@@ -0,0 +1,189 @@
+
+.. code-block:: python
+    LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "verbose": {
+            "format": "{levelname} {asctime} {module} {process:d} {thread:d} {message}",
+            "style": "{",
+        },
+        "simple": {
+            "format": "{levelname} {message}",
+            "style": "{",
+        },
+    },
+    "filters": {
+        "special": {
+            "()": "project.logging.SpecialFilter",
+            "foo": "bar",
+        },
+        "require_debug_true": {
+            "()": "django.utils.log.RequireDebugTrue",
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": "INFO",
+            "filters": ["require_debug_true"],
+            "class": "logging.StreamHandler",
+            "formatter": "simple",
+        },
+        "mail_admins": {
+            "level": "ERROR",
+            "class": "django.utils.log.AdminEmailHandler",
+            "filters": ["special"],
+        },
+    },
+    "loggers": {
+        "django": {
+            "handlers": ["console"],
+            "propagate": True,
+        },
+        "django.request": {
+            "handlers": ["mail_admins"],
+            "level": "ERROR",
+            "propagate": False,
+        },
+        "myproject.custom": {
+            "handlers": ["console", "mail_admins"],
+            "level": "INFO",
+            "filters": ["special"],
+        },
+    },
+}
+
+=====================
+Logging GRPC Requests
+=====================
+
+Django-Socio-GRPC by default log requests only when something goes wrong just like Django does.
+    - If your service raises a GRPCException, it will result by default to a Warning.
+      You can define your own GRPCException and set the log_level as you wish to change this.
+    - If your service raises another Exception, it will result in an Error.
+    - Lastly, if you want to log requests even if they are ok, you can launch django in DEBUG mode
+      or you can enable LOG_OK_RESPONSE in your settings.
+
+Example
+-------
+
+Let's set up a simple logging process and see the results
+
+Here's an example of logging setup.
+
+.. code-block:: python
+    LOGGING = {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "filters": {"require_debug_false": {"()": "django.utils.log.RequireDebugFalse"}},
+        "formatters": {
+            "classic": {
+                "format": "[django]-[%(levelname)s]-[%(asctime)s]-[%(name)s:%(lineno)s] %(message)s"
+            },
+            "fmt": {
+                "format": 'level=%(levelname)s name=%(name)s line=%(pathname)s:%(lineno)s message="%(message)s" socio_service_name="%(socio_service_name)s" socio_action="%(socio_action)s" time=%(asctime)s levelno=%(levelno)s socio_usermanagement_uuid="%(socio_usermanagement_uuid)s" socio_user_roles="%(socio_user_roles)s" socio_client_id="%(socio_client_id)s" funcName=%(funcName)s',
```

Comment:
you have specific socio context in this example. I think fmt format should not be documented in the logging features doc